pallets/bridge/src/lib.rs

```diff
@@ -14,77 +14,6 @@
 //! # Bridge pallet
 //!
 //! This pallet implements a bridge between Chainbridge and Centrifuge Chain.
-//! - [`Config`]
-//! - [`Call`]
-//! - [`Pallet`]
-//!
-//! ## Overview
```

Comment:
I've removed this documentation to ease the maintainability because rustdoc already builds it for you using the inline documentation.